dspace-api/src/main/java/org/dspace/content/Entity.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.content;
+
+import java.util.List;
+
+/**
+ * This class represents an Entity object. An Entity object has an Item that it describes with a list
+ * of relationships that it includes as well.
+ */
+public class Entity {
```

Comment:
This was discussed in the Entity-WG meetings, too. If I remember correctly, this was postponed until after the preview release.

Reply:
ok, so is this class going to be removed? can you please double check that an issue exists on JIRA for that? I'm unable to find it. It will be useful to note in the JIRA issue the decision or link to meeting notes where it was discussed